index.bs

```diff
@@ -1,5 +1,5 @@
 <pre class='metadata'>
-Title: DeviceOrientation Event Specification
+Title: Device Orientation
```

Comment:
We should not rename a web specification that is this established.

There is a large number of other documents (specs, dev docs, articles, etc.) accumulated over the last 13 years that reference this document with its current name and this rename would create more confusion among readers than it would help. As a secondary consideration, with this rename the shortname would be off sync.

If we'd be starting this spec effort now, we'd likely choose a different name, but with the current state of things, I consider this rename not appropriate.

Reply:
Naming things is hard. Renaming things is even harder. I'll defer to the W3C Staff on whether there are any unforeseen issues we'll run into trying to rename an established specification but I'm not immediately opposed to it, but while we're repainting the bike shed... the biggest point of confusion I've found in the old name is that the name ignores that this is document specifies two separate events so I suggest "Device Orientation and Motion Events".

Reply:
We rename specs all the time. It's really not a big deal. Search engines are pretty clever, and so are people. 

I really think this an editors' decision.

Reply:
Bikeshedding is fun. "Device Orientation and Motion Events" is more precise name, if only a bit verbose. "Device Orientation and Motion" as currently proposed has less words and I'm fine with it if it passes the publication checks.